### ISSUE TITLE
php-version has been removed from homebrew

### ISSUE THREAD

**Issue by jonphipps** [wilmoore/php-version#67]
```bash
$ brew tap homebrew/homebrew-php
$ brew install php-version
Error: No available formula with the name "php-version"
==> Searching for a previously deleted formula (in the last month)...
Warning: homebrew/core is shallow clone. To get complete history run:
  git -C "$(brew --repo homebrew/core)" fetch --unshallow

Error: No previously deleted formula found.
==> Searching for similarly named formulae...
==> Searching local taps...
Error: No similarly named formulae found.
==> Searching taps...
==> Searching taps on GitHub...
Error: No formulae found in taps.
```

**Comment by jonphipps** [comment]
It was removed from homebrew-php in this commit;
https://github.com/Homebrew/homebrew-php/commit/3a1e85d0f2b1278df225622acf8766d6686eaea1#diff-39a34f604b24e0756f9d8eaed00c088e

...which just seems crazy and destructive to me. Once a formula is registered, BrewTestBot crawls the referenced repo and updates the formula whenever there’s a new release. So it’s not like there’s a lot of maintenance involved in keeping old formula. 

Sigh.

**Comment by aaron-suarez** [comment]
Apparently, the tap is now deprecated as well:

```bash
$ brew tap homebrew/homebrew-php
Updating Homebrew...
Error: homebrew/php was deprecated. This tap is now empty as all its formulae were migrated.
```

Is anyone maintaining this project anymore? If so, is there an alternative repo to pull from or is building from source the only option?

**Comment by codev0** [comment]
Here is custom [tap](https://github.com/codev0/homebrew-php-version), or use my own.
How to:
1. Fork the repo
2. `brew tap username/reponame` (`homebrew-` prefix is required)
3. `brew install reponame`

I just copied [old tap](https://github.com/Homebrew/homebrew-php/blob/cf7df04d57c491c83a2f9249e6884024b92a1a80/Formula/php-version.rb) and added as own. Brew supports this feature.

**Comment by wilmoore** [comment]
Thanks @codev0 - I had been meaning to get around to this. This prompted me to check out https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md.

I've forked your repo. I have verified that the following works:

```
brew install wilmoore/homebrew-php-version/php-version
```

I'll update the docs.

**Comment by wilmoore** [comment]
Actually, made some changes as I researched a slightly cleaner way to do it. I copied the convention used by https://github.com/thoughtbot/homebrew-formulae.

I also dropped the bottle DSL stuff because it was giving me warnings and seems unnecessary.

https://github.com/wilmoore/homebrew-formulae

**Comment by wilmoore** [comment]
Fixed with a5c4f9f2cd23f09714c2db3480d9a02c63091bc5
